src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobDispatcher.java

```diff
@@ -17,9 +17,7 @@
 
     private final BlobServiceClient bulkscanStorageClient;
 
-    public BlobDispatcher(
-        @Qualifier("bulkscan-storage-client") BlobServiceClient bulkscanStorageClient
-    ) {
+    public BlobDispatcher(BlobServiceClient bulkscanStorageClient) {
```

Comment:
In the next PR BlobDispatcher is going to have client provide injected. I've removed the qualifier, because that client doesn't exist anymore. It wasn't a working client, anyway.